static_src/components/app_container.jsx

```diff
@@ -108,6 +111,28 @@ export default class AppContainer extends React.Component {
     );
   }
 
+  get openApp() {
+    if (AppStore.isRunning(this.state.app)) {
```

Comment:
Assuming you had [two apps bound to the same route](https://github.com/18F/cg-dashboard/pull/980#discussion_r98577410), one might not be running but it could still make sense for the link to appear. I think we should just remove the check and worst case no apps are running and you just get a 404 from Cloud Foundry. Seems reasonable to me.